elconfig.c

```diff
@@ -4161,6 +4325,35 @@ static void elconfig_populate_tabs(void)
 				widget_set_OnKey(window_id, widget_id, (int (*)())spinbutton_onkey_handler);
 				widget_set_OnClick(window_id, widget_id, spinbutton_onclick_handler);
 			break;
+			case OPT_INT_S:
+				label_id = label_add_extended(window_id, elconfig_free_widget_id++, NULL,
+					current_x, current_y, 0, elconf_scale, (char*)var->display.str);
+				widget_width = slider_width;
+				widget_id = slider_add_extended(window_id, elconfig_free_widget_id++, NULL,
+#ifdef ANDROID
+					window_width - right_margin - 2.0f * widget_width, current_y, 2.0f * widget_width, 2.0f * line_height,
```

Comment:
As discussed, the with and x position scaling should be removed to avoid making the widget too wide.